include/wx/generic/splitter.h

```diff
@@ -289,6 +295,8 @@ class WXDLLIMPEXP_CORE wxSplitterWindow: public wxNavigationEnabled<wxWindow>
     wxPoint     m_ptStart;      // mouse position when dragging started
     int         m_sashStart;    // sash position when dragging started
     int         m_minimumPaneSize;
+    int         m_latestHorizontalSashPosition;
+    int         m_latestVerticalSashPosition;
```

Comment:
As mentioned above, I'd use a single `wxPoint` for those and call it `m_lastSplitPosition`.